src/pages/[...slug].tsx

```diff
@@ -149,6 +154,10 @@ export const getStaticProps: GetStaticProps<Props, Params> = async (
     }
   }
 
+  const crowdinContributors = ["docs", "tutorials"].includes(layout ?? "") // TODO: Remove fallback after Layout type implemented (PR #137)
```

Comment:
Should this be tutorial (to match the layout file name)?

```suggestion
  const crowdinContributors = ["docs", "tutorial"].includes(layout ?? "") // TODO: Remove fallback after Layout type implemented (PR #137)
```

I think this is causing `useGitHubContributors` in the `Tutorial.tsx` file to always be true @wackerow @pettinarip 🤔

Reply:
The layout is now called [`tutorial`](https://github.com/ethereum/ethereum-org-fork/blob/dev/src/pages/%5B...slug%5D.tsx#L147) so, this should be fine. Anyway, maybe it is related somehow.

Reply:
@pettinarip just tested locally, this is causing the issue you described above.

Reply:
Ahh great! thanks @minimalsm